microshift_architecture/microshift-understanding.adoc

```diff
@@ -0,0 +1,21 @@
+:_content-type: ASSEMBLY
+[id="microshift-understanding"]
+= Understanding MicroShift
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-understanding
+toc::[]
+
+Learn about {product-title}, interacting with {product-title} the command-line interface (CLI) tools, consumption experience, and integration with {op-system-first}.
```

Comment:
```suggestion
Learn about {product-title}, interacting with the {product-title} command-line interface (CLI) tools, consumption experience, and integration with {op-system-first}.
```
Additionally, what is "consumption experience"? Is this something users are expected to know? It may be worthwhile to elaborate a little more here.

Reply:
There is not a separate command line tool for MicroShift, so I don't think the suggested change is quite what we want.

I know @kalexand-rh has made some changes to the text, but if "interacting with {product-title} the command-line" is from my original version I suspect it was supposed to say "via the command-line".

Reply:
FYI to all, this content has changed in the docs I have worked on. These changes are not reflected here since I was waiting on this PR being conformed to the spreadsheet and merged so I can create new ones.